pkg/cleantemp/cleantemp.go

```diff
@@ -14,11 +14,16 @@ import (
 	logContext "github.com/trufflesecurity/trufflehog/v3/pkg/context"
 )
 
+const (
```

Comment:
@rosecodym - moved the string/formatting to a global const. I think leaving https://github.com/trufflesecurity/trufflehog/pull/2130/files#diff-60ea9ceab29f368c394ac493e6abe8a9195a1fbf7ea2dedbe420f94b8e84cfcfR78 as is may still make sense as it should catch anything with a live pid value in it even if, for whatever reason, it didn't originate from this module.